serialization/src/api/corda-serialization-5.1.0.yaml

```diff
@@ -0,0 +1,123 @@
+net.corda.v5.serialization.MissingSerializerException:
```

Comment:
Some of the things in here are not part of the public api, so I assume the whole module isn't. Is that correct @driessamyn ? If so we can remove the scanning from here.

For example `SingletonSerializeAsToken`.

Reply:
It's a good question. Maybe @josephzunigadaly or @kyriathar may be able to confirm for sure.
On the other hand, I think it's ok to leave it as is, but maybe we need a ticket to move everything that doesn't need to be public API away from this repo instead?

Reply:
Early in Corda 5 development, we went through this module and moved anything that was not public API. What remains here was chosen to be public API at that time.